workspace-manager-clienttests/src/main/resources/serviceaccounts/delegate-user-sa.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "firecloud-dev@broad-dsde-dev.iam.gserviceaccount.com",
```

Comment:
I'm not familiar with this SA, could you give some background on where it comes from/what it's generally used for?

Reply:
Basically, this SA account is used as a user delegate service account for workspace manager service. See `tools.render-config.sh`